### ISSUE TITLE
Rescaling the resulting SVGs? (e.g. zooming in)

### ISSUE THREAD

**Issue by aldanor** [TilBlechschmidt/obsidian-plugin-abcjs#20]
Wonder if there's an easy CSS hack to zoom all of the abc.js output by a certain %? (on some monitors, those can appear quite tiny - and while it's possible to rescale Obsidian's UI font and editor font size, it's not obvious how to do that with abc)

Thanks :)

**Comment by TilBlechschmidt** [comment]
I do not think it is possible with just CSS as it would not change the layout of the sheet and thus either overflow the page boundary (with the edges cut off) or become too crammed (when increasing just the text size). There is a [`scale` option](https://paulrosen.github.io/abcjs/visual/render-abc-options.html#scale) for abc.js but at least in the Obsidian context it does not appear to do anything:
``````
```music-abc
{
  "scale": 2
}
---
X:1
T: Cooley's
M: 4/4
L: 1/8
R: reel
K: G
|:D2|EB{c}BA B2 EB|~B2 AB dBAG|FDAD BDAD|FDAD dAFD|
```
``````
Though that might be an avenue to explore further. Maybe a combination of the other options might work too.

**Comment by aldanor** [comment]
Hmm, looks like removing/overriding `"resize":"responsive"` allows scale to work – but that's not really the same thing, because as you said - it gets cut on the right side boundary if there's not enough space.

It would be nice to keep it responsive but just give it a bigger "maximum normal size", wherever that is actually defined...

<img width="757" alt="image" src="https://user-images.githubusercontent.com/2418513/196277715-29c94757-5448-46b1-8200-4f1473dcdd6b.png">

**Comment by paulrosen** [comment]
The "responsive" item makes the music as large as possible to fit the surrounding container, so making the surrounding container larger will make the music larger. And it isn't compatible with zoom because it will override whatever zoom you've picked. You can change the `%%staffwidth`. If the width is smaller then the responsive setting will make the whole thing bigger. You could also play around with wrapping if staffwidth almost works but makes the music too crowded.
